messaging/src/main/java/org/axonframework/eventhandling/scheduling/quartz/QuartzEventScheduler.java

```diff
@@ -340,15 +340,14 @@ private Instant retrieveDeadlineTimestamp(JobDataMap jobDataMap) {
     /**
      * Builder class to instantiate a {@link QuartzEventScheduler}.
      * <p>
-     * The {@link EventJobDataBinder} is defaulted to an {@link DirectEventJobDataBinder}, and the
-     * {@link TransactionManager} defaults to a {@link NoTransactionManager}.
+     * The {@link TransactionManager} defaults to a {@link NoTransactionManager}.
      * The {@link Scheduler} and {@link EventBus} are a <b>hard requirements</b> and as such should be provided.
      */
     public static class Builder {
 
         private Scheduler scheduler;
         private EventBus eventBus;
-        private EventJobDataBinder jobDataBinder = new DirectEventJobDataBinder();
+        private EventJobDataBinder jobDataBinder;
```

Comment:
Changing this to a `Supplier<EventJobDataBinder>` should have the same effect.

You'd thus in the constructor invoke `jobDataBinder.get()`, invoking the creation of the `EventJobDataBinder` (with the defaulted `XStreamSerializer`) at that stage, instead of always.

Reply:
That is a better approach, I've updated.